src/openrct2/ride/Vehicle.cpp

```diff
@@ -2690,15 +2690,15 @@ static rct_synchronised_vehicle* _lastSynchronisedVehicle = nullptr;
  * to synchronise to the vehicle synchronisation list.
  *  rct2: 0x006DE1A4
  */
-static bool try_add_synchronised_station(int32_t x, int32_t y, int32_t z)
+static bool try_add_synchronised_station(const CoordsXYZ& coords)
```

Comment:
This function is slightly wrong the coordinates of z is in small Z and not big Z. We need to change all callers and functions that this calls.

Reply:
Fixed on #12192